### ISSUE TITLE
Import to Magento 2 with configurable Products?

### ISSUE THREAD

**Issue by kingalca** [dweeves/magmi-git#413]
Hi, can Magmi import Products successfully into Magneto 2?

**Comment by Phizik-PRO** [comment]
I subscribe to the issue

**Comment by PedroDiaz** [comment]
Me too, there is a conversation on issue  #369 where @tagesjump has something advanced

**Comment by sammarcus** [comment]
@kingalca No, in its current state it cannot handle configurable products in Magento 2.

**Comment by kordzadeh** [comment]
any news about configurable products?
i can import simple products very well and also categories and his values on the fly.
but unfortunately have big problem on configurable products. is any body imports configurable product in magento 2 by magmi?
